pkg/generator/helper/helper.go

```diff
@@ -115,6 +120,39 @@ func Test{{.CobraCmdFuncName}}(t *testing.T) {
 }
 `
 
+const enumGoTemplate = `package cmd
+
+import (
+	"fmt"
+	"errors"
+)
+type {{.Name}} int
+
+const (
+	{{$enumName := .Name}}
+	UNKNOWN_{{$enumName | toUpperCase}} {{$enumName}} = iota -1
+	{{- range $value := .Values}}
+	{{$value}} {{$enumName}} = iota{{end}}
+)
+
+func (w {{.Name}}) String() string {
+	return [...]string{
+		"UNKNOWN_{{.Name | toUpperCase}}",
+		{{- range $value := .Values}}
+		"{{$value}}",{{end}}
+	}[w]
+}
+
+func ValueOf(str string) ({{.Name}}, error) {
```

Comment:
This does not work. Starting with the second enum we will have a clash for the ValueOf function for all the enums. Maybe namespaced ...